instrumentation/grails-3.0/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/grails/DefaultGrailsControllerClassInstrumentation.java

```diff
@@ -14,9 +14,9 @@
 
```

Comment:
we also need to update the [module](https://github.com/SylvainJuge/opentelemetry-java-instrumentation/blob/indy-grails/instrumentation/grails-3.0/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/grails/GrailsInstrumentationModule.java#L16) with `ExperimentalInstrumentationModule` and `isIndyReady()`